### PR TITLE
Fix panic related to empty yaml config

### PR DESCRIPTION
**Pull request by MirkoCovizzi**
This solves an `index out of bounds` panic.

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@ use crate::tile_grid::SplitDirection;
 use std::io::{Error, ErrorKind};
 
 use std::str::FromStr;
+use yaml_rust::{ScanError, Yaml};
```
Comment (TimUntersberger) [inline src/config.rs:6]: Why do we need the ScanError here?

```suggestion
use yaml_rust::{Yaml};
```

File: src/config.rs [hunk 2]
```diff
@@ -64,7 +65,13 @@ pub fn load() -> Result<Config, Box<dyn std::error::Error>>{
     };
 
     let file_content = std::fs::read_to_string(path).unwrap();
-    let yaml = &yaml_rust::YamlLoader::load_from_str(&file_content).unwrap()[0];
+
+    let vec_yaml = yaml_rust::YamlLoader::load_from_str(&file_content).unwrap();
+    let mut yaml = &Yaml::Null;
```
Comment (TimUntersberger) [inline src/config.rs:70]: Overall I'd like to keep the style the same so either every `yaml_rust::......` usage has to change or we rewrite it to use namespaces

```suggestion
    let mut yaml = &yaml_rust::Yaml::Null;
```

**Comment by TimUntersberger** [comment]
Besides there 2 comments everything is looking good. Thank you!

**Comment by MirkoCovizzi** [comment]
Done and done

**Pull request merged**